From ebf8ce137a9b8dea49c3c4fbcfeeb2b3a1ffb35b Mon Sep 17 00:00:00 2001
From: Steve deRosier <steve.derosier@lairdtech.com>
Date: Sat, 5 Dec 2015 12:20:59 -0800
Subject: [PATCH] laird: wb45n: move the pmecc_header file to reflect the ECC
 strength

The wb45 and wb50 will have different ECC strengths, so we will need header
files specific to that strenght. Pave the way by changing the name.

Bug: 8689
---
 board/at91sam9x5ek/board.mk | 2 +-
 board/pmecc_header.bin      | 1 -
 board/pmecc_header4.bin     | 1 +
 3 files changed, 2 insertions(+), 2 deletions(-)
 delete mode 100644 board/pmecc_header.bin
 create mode 100644 board/pmecc_header4.bin

diff --git a/board/at91sam9x5ek/board.mk b/board/at91sam9x5ek/board.mk
index 83985cf..fb9e4cc 100644
--- a/board/at91sam9x5ek/board.mk
+++ b/board/at91sam9x5ek/board.mk
@@ -7,5 +7,5 @@ ASFLAGS += \
 	-mcpu=arm926ej-s
 
 ifeq ($(CPU_HAS_PMECC),y)
-	PMECC_HEADER := "board/pmecc_header.bin"
+	PMECC_HEADER := "board/pmecc_header4.bin"
 endif
diff --git a/board/pmecc_header.bin b/board/pmecc_header.bin
deleted file mode 100644
index f8d6073..0000000
--- a/board/pmecc_header.bin
+++ /dev/null
@@ -1 +0,0 @@
-$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿
\ No newline at end of file
diff --git a/board/pmecc_header4.bin b/board/pmecc_header4.bin
new file mode 100644
index 0000000..f8d6073
--- /dev/null
+++ b/board/pmecc_header4.bin
@@ -0,0 +1 @@
+$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿$ê¿
\ No newline at end of file
-- 
1.9.1

